sysdig/resource_sysdig_secure_managed_policy_test.go

```diff
@@ -107,22 +107,19 @@ resource "sysdig_secure_managed_policy" "sample" {
 		  name = "testcapture"
 		}
 	}	
-}
-	`)
+}`
 }
 
 func managedPolicyWithMinimumConfiguration() string {
-	return fmt.Sprintf(`
+	return `
 resource "sysdig_secure_managed_policy" "sample" {
 	name = "Sysdig Runtime Threat Detection"
 	enabled = true
-}
-	`)
```

Comment:
The same here

Reply:
Thats ok because `fmt.Sprintf` is removed since linter is failing after modifying `golangci-lint` command. Check here: https://github.com/sysdiglabs/terraform-provider-sysdig/pull/349/files/9d3c9ba36d3d4e691327a43d5c2b102c4d100ef9#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R18

Reply:
Yup. Thanks!